src/MDMSwiftyObjectiveC.h

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Naming is hard: Why is this call "swifty" objectivec? It seems to be more like a "dontsubclass.h"...

Reply:
Haha because this is a feature from Swift. I don't feel strongly about the name, will change to something more direct.

Reply:
Done.